src/core/index.js

```diff
@@ -10,8 +10,6 @@ const cache = new Cache(session)
 const { navigator } = session
 export { navigator, session, cache, PageRenderer, PageSnapshot, FrameRenderer }
 
-export { StreamActions } from "./streams/stream_actions"
```

Comment:
Can we keep the export here too? It's not strictly necessary, but it will avoid breaking libraries and client code that depend on this import.

Reply:
I've pushed up a change to continue to export as `Turbo.StreamActions`.